config/env.development

```diff
@@ -187,22 +187,6 @@ DEPENDENCY_DISCOVERY_URL=http://localhost/v1/dependency-discovery
 # NODE_ENV should be one of "development" or "production"
```

Comment:
I wonder if we can remove this entire `Telescope 1.0 Legacy Environment` block?  Is anything in here needed?

Reply:
Parser still uses these ones 
```yml
      - NODE_ENV
      - PARSER_PORT
      - ELASTIC_MAX_RESULTS_PER_PAGE
      - ELASTIC_URL=http://elasticsearch
      - ELASTIC_PORT=9200
```

Reply:
Seems like only `parser` uses those variables. I can do some detective work, move in-use variables to parser section